internal/highlight/highlight.go

```diff
@@ -372,3 +388,62 @@ func generatePlainTable(code string) (template.HTML, error) {
 	}
 	return template.HTML(buf.String()), nil
 }
+
+// unhighlightLongLines takes highlighted HTML and unhighlights lines which are
+// longer than N bytes in (plaintext) length, making them easier for some
+// browsers such as Chrome to render.
+//
+// And the returned HTML has all <span> tags removed from lines whose length
+// are > N bytes.
+//
+// See https://github.com/sourcegraph/sourcegraph/issues/6489
+func unhighlightLongLines(h string, n int) (string, error) {
+	doc, err := html.Parse(strings.NewReader(h))
+	if err != nil {
+		return "", err
+	}
+
+	table := doc.FirstChild.LastChild.FirstChild // html->body->table
```

Comment:
Tiny suggestion, this way this is actually a well-defined CSS selector :)
```suggestion
	table := doc.FirstChild.LastChild.FirstChild // html > body > table
```

Reply:
Thanks, looks much better!